### ISSUE TITLE
Unable to type in windows app after clicking on notification

### ISSUE THREAD

**Issue by sah4jpatel** [BlueBubblesApp/bluebubbles-app#2656]
Been having an issue for a little while where when I click on a windows bluebubbles notification, the text box does not register input and I usually have to back out to the chat list and then open up that chat again.

I am on v1.12.7 on Windows 11 23H2 using the app using the .exe installer (not windows store) with the app window aspect ratio being closer to candy bar iPhone shape.

Minor inconvenience for sure, but would be good to have addressed (even if its user error) eventually.

**Comment by sp1nkick** [comment]
I'm seeing this too

**Comment by sah4jpatel** [comment]
An Additional detail: the text is there that is type but it is not visible. I have to back out to the chat list and then back into the chat and then it is visible.

**Comment by sah4jpatel** [comment]
Ill give it a shot rn

**Comment by jjoelj** [comment]
Did that resolve the issue?

**Comment by sah4jpatel** [comment]
I don't think so. Started experiencing it again on a fresh install where when the window is open and then I click out into a different window to do something else and then click back into the bluebubbles text box, it doesn't register key strokes until I keep clicking (rarely works) or just back out into the chat list and the go back. Happy to keep testing, its not as common an issue anymore, but still happens occasionally.

**Comment by sah4jpatel** [comment]
The issue now mainly and consistently occurs when the taskbar icon is clicked to either bring the app from the background to the front, or to unminimize it. Slightly different behavior when minimize to tray is enabled. The text cursor blinks but no text is registered usually, sometimes it appears 15 seconds later but its not usable until i go back to the chat list  and go into the chat. In the minimize to taskbar mode, the cursor doesnt blink and the textbox isnt clickable.

**Comment by jjoelj** [comment]
@sah4jpatel can you confirm whether this issue is still present in the latest beta?
